actions/helm_generate.go

```diff
@@ -31,6 +32,9 @@ func generateParams(stage bool, fs fileSys, whereTo string, paramsComponentMap g
 			log.Fatalf("Error creating staging file (%s)", err)
 		}
 	}
+	if strings.Contains(whereTo, "e2e") {
+		return generateParamsE2ETpl.Execute(executeTo, paramsComponentMap)
+	}
```

Comment:
this is a little weird to choose the template based on the `whereTo` string. maybe a better way is to have the `generateParam` func take in a `*text.Template`. thoughts?